src/AutoChessSharp.Core/Controller/GameRunner.cs

```diff
@@ -35,7 +35,7 @@ public GameRunner(IBoard board)
         _store = new Store();
         _store.RerollStore();
         _gameStatus = GameStatusEnum.NotStarted;
-        _playerDetail = new Dictionary<IPlayer, PlayerInfo>();
+        _playerDetail = new Dictionary<IPlayer, IPlayerInfo>();
```

Comment:
fundamental fields changed to receive interfaces as their values / members to provide extensibility